include/loramesh.h

```diff
@@ -14,6 +16,10 @@ class LoraMesh {
   DataQueue<message_t>* txQueue;
   DataQueue<message_t>* rxQueue;
   Router* router;
+  Layer1Class* Layer1;
+  LL2Class* LL2;
+  char routes[256];
+  char neighbors[256];
```

Comment:
Both routes and neighbors are not used, thus not necessary. Please remove it.

Reply:
Essas variáveis foram declaradas por um momento que precisamos, mas não precisamos mais delas. Thanks